commands_add.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type addCmd struct {
-	id     uint64
+	txID   uint64
```

Comment:
I feel that this needs to be changed globally.  It is good naming convention.  We should instantly know from the name of the variable which ID this is referring to.